package.json

```diff
@@ -13,6 +13,7 @@
   },
   "homepage": "https://github.com/chas-academy/project-api-boilerplate/tree/master",
   "dependencies": {
+    "algoliasearch": "^3.27.1",
```

Comment:
I've checked out your branch and `yarn install`, though this message keeps popping up as I execute `bin/start` at the root of `api`. 
![screenshot from 2018-05-12 15-06-52](https://user-images.githubusercontent.com/31239061/39957638-7eab31aa-55f6-11e8-8ec2-048af39f2164.png)